### PR TITLE
Users/vinca/bugfix

### PR DESCRIPTION
**Pull request by vincentdass**
Replacing **".\username"** by **"MachineName\username"**.
Tested with both domain and workgroup.
Patch Version Incremented.
Gulp tests passed.

### CODE REVIEW

File: Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1 [hunk 1]
```diff
@@ -75,7 +75,19 @@ param (
             ThrowError -errorMessage (Get-LocalizedString -Key "Parameter '{0}' cannot be null or empty." -ArgumentList "credential")
         }   
     }
-    
+
+    function Get-FullUserName(
```
Comment (kmkumaran) [inline Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1]: Can you name this function Get-DownLevelLogonName... Refer: https://msdn.microsoft.com/en-us/library/windows/desktop/aa380525(v=vs.85).aspx.

**Comment by kmkumaran** [comment]
Looks fine.

**Comment by KrishnaAdityaB** [comment]
Vincent, 
I think these code changes don't support the case where FQDN of the machine is given as - mymachinename.fareast.corp.microsoft.com. Can you quickly verify?

**Comment by vincentdass** [comment]
Hi Krishna,
Domain and Local user credentials tests passed.
Gulp tests passed

**Pull request merged**